ipc/src/stream_codec.rs

```diff
@@ -0,0 +1,137 @@
+extern crate tokio_core;
+
+use std::io;
+use stream_codec::tokio_core::io::{Codec, EasyBuf};
+
+pub struct StreamCodec;
+
+fn is_whitespace(byte: u8) -> bool {
+	match byte {
+		0x0D | 0x0A | 0x20 | 0x09 => true,
```

Comment:
so we don't trim anything, we just don't consider some requests as a compete ones until there is at least one non-whitespace character in it